src/simulator/render.py

```diff
@@ -71,10 +71,8 @@ def render_vehicles(screen: Surface, vehicles: list[Vehicle]) -> None:
         if vehicle_center_point is None: # this vehicle is out of its route and returns no position
             continue
         vehicle_center_screen_pos = world_to_screen_vector(screen, vehicle_center_point, zoom_factor)
-        img = pygame.transform.smoothscale(vehicle.image, (vehicle_screen_length, vehicle_screen_width))
-        vehicle_angle = direction_at_route_position(vehicle.route, vehicle.route_position)
-        vehicle.direction_angle = vehicle_angle
-        img = pygame.transform.rotate(img, vehicle_angle)
+        imgNoRotation = pygame.transform.smoothscale(vehicle.image, (vehicle_screen_length, vehicle_screen_width))
```

Comment:
we want our variables to have words separated by underscore (snake_case). This is the python standard for variables